Tests/InstallationTests/ReceiptParserInstallation/ReceiptParserIntegrationTests/ReceiptParserIntegrationTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ReceiptParserIntegrationTests.swift
+//  ReceiptParserIntegrationTests
+//
+//  Created by Nacho Soto on 11/29/22.
+//
+
+import ReceiptParser
+import ReceiptParserIntegration
+import XCTest
+
+final class ReceiptParserIntegrationTests: XCTestCase {
```

Comment:
Should we add a success test case here?

Reply:
I'd need to have a sample receipt in here, which [are pretty large](https://github.com/RevenueCat/purchases-ios/tree/4.15.4/Tests/UnitTests/Resources/receipts).
The actual logic for parsing receipts is already tested in unit tests. This is an _installation_ test, so I think simply checking that the method can be called and it does what it's supposed to is enough.